test/test_type_hints.py

```diff
@@ -229,7 +203,6 @@ def test_run_mypy_strict(self):
 
         with set_cwd(repo_rootdir):
             (stdout, stderr, result) = mypy.api.run([
```

Comment:
interesting that mypy.strict never specified `--check-untyped-defs`, but defined in mypy-strict.ini.